fmn/rules/utils.py

```diff
@@ -67,6 +72,32 @@ def get_fas(config):
     return _FAS
 
 
+def _paginate_pagure_data(url, params, key):
```

Comment:
@jeremycline I think it'd be helpful to have a shared library, but that's really a separate discussion. Can we modify FMN to use the library once it has been written, so it doesn't block this PR?